models/ipns.go

```diff
@@ -99,6 +115,7 @@ func (im *IpnsManager) CreateEntry(ipnsHash, ipfsHash, key, networkName string,
 	entry.TTL = ttl.String()
 	entry.Key = key
 	entry.NetworkName = networkName
+	entry.UserName = username
```

Comment:
instead of a bunch of `entry.___`, why not just `entry := IPNS{Sequence: 1, ...}`

Reply:
good spot!